### ISSUE TITLE
ImportError: No module named 'pyembed.markdown.extension'

### ISSUE THREAD

**Issue by lanox** [pyembed/pyembed#57]
Hi I was wondering is this working with python 3.4 ? I seem to be getting following error 

```
pelican -r /Users/xxxx/Sites/pelican_blog/content -o /Users/xxxx/Sites/pelican_blog/output -s /Users/xxx/Sites/pelican_blog/pelicanconf.py
Traceback (most recent call last):
  File "/Users/xxx/.pyenv/versions/my_blog/bin/pelican", line 11, in <module>
    sys.exit(main())
  File "/Users/xxxx/.pyenv/versions/my_blog/lib/python3.4/site-packages/pelican/__init__.py", line 351, in main
    pelican, settings = get_instance(args)
  File "/Users/xxx/.pyenv/versions/my_blog/lib/python3.4/site-packages/pelican/__init__.py", line 337, in get_instance
    settings = read_settings(config_file, override=get_config(args))
  File "/Users/xxxx/.pyenv/versions/my_blog/lib/python3.4/site-packages/pelican/settings.py", line 145, in read_settings
    local_settings = get_settings_from_file(path)
  File "/Users/xxx/.pyenv/versions/my_blog/lib/python3.4/site-packages/pelican/settings.py", line 197, in get_settings_from_file
    module = load_source(name, path)
  File "/Users/xxxxx/.pyenv/versions/my_blog/lib/python3.4/site-packages/pelican/settings.py", line 14, in <lambda>
    load_source = lambda name, path: SourceFileLoader(name, path).load_module()
  File "<frozen importlib._bootstrap>", line 539, in _check_name_wrapper
  File "<frozen importlib._bootstrap>", line 1614, in load_module
  File "<frozen importlib._bootstrap>", line 596, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 1220, in load
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/Users/xxxx/Sites/pelican_blog/pelicanconf.py", line 4, in <module>
    from pyembed.markdown.extension import PyEmbedMarkdown
ImportError: No module named 'pyembed.markdown.extension'
make: *** [regenerate] Error 1
```

**Comment by matt-thomson** [comment]
A couple of things:

* If you want to use the Markdown extension, you need to install the `pyembed-markdown` package.
* There's no `pyembed.markdown.extension` package - the import statement should be:

```python
from pyembed.markdown import PyEmbedMarkdown
```

This was previously incorrect in some of the [documentation](http://pyembed.github.io/usage/markdown/) - I've fixed the incorrect references.

Let me know how you get on!

**Comment by lanox** [comment]
@matt-thomson Thanks for the quick reply... That seem to fix the issue, however i am experiencing weird behaviour where make regenerate takes 
```
Done: Processed 7 article(s), 0 draft(s) and 2 page(s) in 22.79 seconds.
```

Also it breaks pygment highlighting and creating tables with markdown.

But if i remove 
```
from pyembed.markdown import PyEmbedMarkdown
MD_EXTENSIONS = [PyEmbedMarkdown()]
``` 
from pelicanconf.py all works as normal. 

I am not sure if i gave you enough info here but let me know what you need and happy to provide.

Cheers

**Comment by matt-thomson** [comment]
The embedding does take some time, as the extension needs to make a number of HTTP requests (this is inherent in the way that OEmbed works).  I've a few ideas as to how these can be reduced and sped up - hope to take a look in the next couple of weeks.

I'm not aware of any issues with Pygments or tables.  If you could post an example Markdown file that renders incorrectly I'd be happy to take a look.

**Comment by lanox** [comment]
@matt-thomson let me try it again and i will get back to you . Thanks for all the help.

**Comment by matt-thomson** [comment]
I've released v1.3.0 today with the discovery improvements.  You should hopefully see a performance improvement if you upgrade - I'd be interested to hear how you get on.

Going to close this issue now, but if you do reproduce the Pygments/tables issue, then please report it at [pyembed-markdown](https://github.com/pyembed/pyembed-markdown).

**Comment by lanox** [comment]
thanks @matt-thomson, will give it a try
